src/pages/integrations/infrastructure-metrics/protocols/metricbeat-module-http.mdx

```diff
@@ -5,7 +5,7 @@ subTitle: Collect and ship HTTP endpoint server metrics to Logstash and Elastics
 logo: https
 color: "#30a743"
 description: How to send HTTP endpoint server metrics to your Hosted ELK Logstash instance. Configure Metricbeat to send HTTP endpoint server metrics to Elasticsearch.
-stackTypes: metrics
+stackTypes: logs
```

Comment:
@pwatson-logit This isnt correct, its still metrics? we can discuss?

Reply:
If we set stackTypes to be metrics we get the vm stacks details rather than the ES stack details. So while it may be metrics data the stack type as we define it is logs.